tests/templates/kuttl/opa/40_superset.yaml.j2

```diff
@@ -0,0 +1,42 @@
+# $NAMESPACE will be replaced with the namespace of the test case.
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: superset-credentials
+type: Opaque
+stringData:
+  adminUser.username: admin
+  adminUser.firstname: Superset
+  adminUser.lastname: Admin
+  adminUser.email: admin@superset.com
+  adminUser.password: admin
+  connections.secretKey: aQC11KVUJ3yTVcy2
+  connections.sqlalchemyDatabaseUri: postgresql://superset:superset@superset-postgresql/superset
+---
+apiVersion: superset.stackable.tech/v1alpha1
+kind: SupersetCluster
+metadata:
+  name: superset
+spec:
+  image:
+    custom: docker.stackable.tech/stackable/superset:4.0.2-stackable0.0.0-dev-opa
+    productVersion: "{{ test_scenario['values']['superset'] }}"
+    pullPolicy: IfNotPresent
+  clusterConfig:
+    authorization:
+      opa:
+        configMapName: opa
+        package: superset
+        cache: {}
```

Comment:
I think most users will not tweak the cache, so it's good to test the happy path I guess
```suggestion
```

Reply:
No idea why this was closed ^^